### DISCUSSION TITLE
Wrangling Multivariate Data

### DISCUSSION THREAD

**Discussion by ebagjones** [timeseriesAI/tsai#265] [Q&A] [answered]
I'm having fun playing around with tsai, but so far I'm getting pretty poor results which I suspect could be related to me screwing up how I have transformed the data. 
A few questions:
1. 
When data for a feature is arranged in a row, does the left-most column (0) represent the last (temporally) piece of data, or the first? 
e.g. https://colab.research.google.com/github/timeseriesAI/tsai/blob/master/nbs/011_data.preparation.ipynb#scrollTo=qZhgBeEUQDww, is the value in 1 measured before or after 6?

2. I saw in March there were issues observed with ordering of data with multivariate sets and df2xy. Are these issues fixed in v0.2.22?

**Comment by oguiza** [comment, marked as answer]
Hi @jefferyanderson,

1. Column 0 represents the first (oldest) value. 
2. Yes, those were fixed.

**Reply by ebagjones** [reply]
Thanks @oguiza much appreciated.
